### PR TITLE
Hyundai: move Optima Hybrid to dashcam only

### PR DESCRIPTION
**Pull request by sshane**
No routes in the last year, and it's not in CARS.md

### CODE REVIEW

File: selfdrive/car/hyundai/interface.py [hunk 1]
```diff
@@ -175,7 +176,7 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[]):  # py
       ret.lateralTuning.indi.timeConstantV = [1.4]
       ret.lateralTuning.indi.actuatorEffectivenessBP = [0.]
       ret.lateralTuning.indi.actuatorEffectivenessV = [1.8]
-    elif candidate in (CAR.KIA_OPTIMA, CAR.KIA_OPTIMA_H):
+    elif candidate == CAR.KIA_OPTIMA:
```
Comment (adeebshihadeh) [inline selfdrive/car/hyundai/interface.py:179]: I would still keep it here

File: selfdrive/car/hyundai/interface.py [hunk 2]
```diff
@@ -29,6 +29,7 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[]):  # py
     ret.openpilotLongitudinalControl = Params().get_bool("DisableRadar") and (candidate not in LEGACY_SAFETY_MODE_CAR)
 
     ret.pcmCruise = not ret.openpilotLongitudinalControl
+    ret.dashcamOnly = candidate == CAR.KIA_OPTIMA_H
```
Comment (adeebshihadeh) [inline selfdrive/car/hyundai/interface.py:32]: add same comment as GM

Comment (sshane) [inline selfdrive/car/hyundai/interface.py:32]:
```suggestion
    # These cars have been put into dashcam only due to both a lack of users and test coverage.
    # These cars likely still work fine. Once a user confirms each car works and a test route is
    # added to selfdrive/test/test_routes, we can remove it from this list.
    ret.dashcamOnly = candidate in [CAR.KIA_OPTIMA_H, CAR.ELANTRA_GT_I30]
```

File: selfdrive/car/hyundai/interface.py (hunk 1 above)
Comment (sshane) [inline selfdrive/car/hyundai/interface.py:179]:
```suggestion
    elif candidate in (CAR.KIA_OPTIMA, CAR.KIA_OPTIMA_H):
```

**Comment by sshane** [comment]
Woops, the branch is named dashcam so I can't push to it anymore.

**Pull request closed without merging**